src/js/containers/bulkDownload/BulkDownloadPageContainer.jsx

```diff
@@ -127,6 +127,15 @@ export class BulkDownloadPageContainer extends React.Component {
             }
         }
 
+        if (formState.agencyType) {
+            delete params.filters.agency;
+            const agencyParams = { type: formState.agencyType === 'awarding_agency' ? 'awarding' : 'funding', tier: "toptier", name: formState.agency.name };
+            if (formState.subAgency.name) {
```

Comment:
When testing locally this logic is causing problems. If I select a toptier agency, but no subtier agency you still have `Select a Sub-Agency` as the value of `formState.subAgency.name`. This means that an agency such as `Department of Agriculture` is being search on as a subtier agency when they are toptier. 

Additionally, I don't think you are filtering on both toptier and subtier agency. When I try to filter on both the `agencies` filter only has one item and it is the toptier agency listed as a subtier. You will need to handle the filter similar to Advanced Search (link for reference: https://www.usaspending.gov/#/search/96a36d9a791f25cf2cbf0bb75bc14533).